packages/story-editor/src/components/videoTrim/trimmerComponents.js

```diff
@@ -88,11 +88,11 @@ export const ButtonWrapper = styled.div`
   justify-content: ${({ isStart }) => (isStart ? 'flex-end' : 'flex-start')};
 `;
 
-export function Scrim({ width, atStart = false }) {
-  const floatStyle = atStart ? { left: '-1px' } : { right: '-1px' };
+export function Scrim({ width, isLeftAligned = false }) {
```

Comment:
I renamed this as part of changing the direction of tape play for RTL, but then reversed all that again. I kept the renaming as it was clearer though.